### PR TITLE
[feature](merge-cloud) Disable not supported stmt in cloud mode

### PR DESCRIPTION
**Pull request by swjtu-zhanglei**
* In cloud mode, node is managed by meta-servcie restful api not sql stmt
  so alter system stmt will not work

### CODE REVIEW

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/qe/DdlExecutor.java [hunk 1]
```diff
@@ -293,6 +294,11 @@ public static void execute(Env env, DdlStmt ddlStmt) throws Exception {
         } else if (ddlStmt instanceof AdminCompactTableStmt) {
             env.compactTable((AdminCompactTableStmt) ddlStmt);
         } else if (ddlStmt instanceof AdminSetConfigStmt) {
+            if (Config.isCloudMode()
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/qe/DdlExecutor.java:297]: Do stmts check when handleQueryStmt() to decrease the intrusion.

**Review by gavinchou — CHANGES_REQUESTED** [review]
more stmts should be considered

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
> more stmts should be considered

done

**Review by gavinchou — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**